less/index.less

```diff
@@ -52,8 +52,209 @@ table {
 * {
     box-sizing: border-box;
 }
+/**** Variables ****/
+
+@defaultbg: #808080;
```

Comment:
Good job with the variables and mixin's (I know it can be a bit annoying making/using them on such a relatively small project, but it's a skill you'll be super thankful for once you move on big projects where things are thousands upon thousands of lines of code, anything that can save you even a bit of typing at that point is a god-send). Good job showing you understand the material and concepts.